tests/test_healpix_ffts.py

```diff
@@ -92,3 +93,94 @@ def test_healpix_ifft_cuda(flm_generator, nside):
         atol=1e-7,
         rtol=1e-7,
     )
+
+
+@pytest.mark.skipif(not gpu_available, reason="GPU not available")
+@pytest.mark.parametrize("nside", nside_to_test)
+def test_healpix_fft_cuda_transforms(flm_generator, nside):
+    L = 2 * nside
+
+    # Generate a random bandlimited signal
+    def generate_flm():
+        flm = flm_generator(L=L, reality=False)
+        flm_hp = samples.flm_2d_to_hp(flm, L)
+        f = hp.sphtfunc.alm2map(flm_hp, nside, lmax=L - 1)
+        return f
+
+    f_stacked = jnp.stack([generate_flm() for _ in range(10)], axis=0)
+
+    def healpix_jax(f):
+        return healpix_fft_jax(f, L, nside, False).real
+
+    def healpix_cuda(f):
+        return healpix_fft_cuda(f, L, nside, False).real
+
+    f = f_stacked[0]
+    # Test VMAP
+    assert_allclose(
+        jax.vmap(healpix_jax)(f_stacked),
+        jax.vmap(healpix_cuda)(f_stacked),
+        atol=1e-7,
+        rtol=1e-7,
+    )
+    # test jacfwd
+    assert_allclose(
+        jax.jacfwd(healpix_jax)(f),
+        jax.jacfwd(healpix_cuda)(f),
+        atol=1e-7,
+        rtol=1e-7,
+    )
+    # test jacrev
+    assert_allclose(
+        jax.jacrev(healpix_jax)(f),
+        jax.jacrev(healpix_cuda)(f),
+        atol=1e-7,
+        rtol=1e-7,
+    )
+
+
+@pytest.mark.skipif(not gpu_available, reason="GPU not available")
+@pytest.mark.parametrize("nside", nside_to_test)
+def test_healpix_ifft_cuda_transforms(flm_generator, nside):
+    L = 2 * nside
+
+    # Generate a random bandlimited signal
+    def generate_flm():
+        flm = flm_generator(L=L, reality=False)
+        flm_hp = samples.flm_2d_to_hp(flm, L)
+        f = hp.sphtfunc.alm2map(flm_hp, nside, lmax=L - 1)
```

Comment:
I think we could use `s2fft.inverse(flm, L=L, reality=False, method="jax", sampling="healpix")` here instead of going via `healpy`? Rationale being that I would have a slight preference for minimising the number of additional tests that depend on `healpy` as it we are no longer requiring it as direct dependency for package and in the long run it might be possible to also remove it as a test dependency.